geth/api/backend.go

```diff
@@ -175,6 +176,27 @@ func (b *StatusBackend) ResetChainData() error {
 	return b.startNode(&newcfg)
 }
 
+func (m *StatusBackend) SeedWhisperSymKey() (string, string, error) {
+	// generate mnemonic phrase
+	mn := extkeys.NewMnemonic(extkeys.Salt)
+	mnemonic, err := mn.MnemonicPhrase(128, extkeys.EnglishLanguage)
+	if err != nil {
+		return "", "", err
+	}
+
+	whisperService, err := m.nodeManager.WhisperService()
+	if err != nil {
+		return "", "", err
+	}
+
+	skid, err := whisperService.AddSymKeyFromPassword(mnemonic)
```

Comment:
this function should not have any side effects on whisper or any other state

Reply:
just to clarify what i mean, there should be only one channel for making changes to internal state of whisper and this channel is `shh` api